lib/node_modules/@stdlib/math/base/special/ceiln/src/ceiln.c

```diff
@@ -32,8 +32,9 @@
 
 // VARIABLES //
 
+#define HUGE 1.0e308
```

Comment:
What is the advantage of doing this? Why isn't the previous static const declaration sufficient?

Reply:
Previous one throws an error

Reply:
The solution then is to rename the original variable, not overwrite the built-in macro. E.g., `static const HUGE_VALUE` or something similar.

Reply:
Yup perfect.